include/picongpu/param/incidentField.param

```diff
@@ -95,23 +114,33 @@ namespace picongpu
                 }
             };
 
-            /** Source of incident E and B fields.
+            /** Make a profile with the user-provided functors defined above.
              *
-             * Each source type combines functors for incident field E and B, which must be consistent to each other.
+             * The functors for incident field E and B must be consistent to each other.
+             * The profile can be used for XMin, XMax, etc.
              */
-            using MySource = Source<FunctorIncidentE, FunctorIncidentB>;
+            using MyProfile = profiles::Free<UserFunctorIncidentE, UserFunctorIncidentB>;

```

Comment:
Offline discussed: It is somehow unclear when we need to provide the trait `GetFunctorIncidentE` for a user incident field profile and when not.

Reply:
Yes, so I will rethink this part and try to simplify it it possible.

Reply:
Indeed, upon thinking more I came to a conclusion this trait is only needed internally, but not to be exposed to a user. And my hypothetical scenario which caused to expose it initially is actually better implemented on a user side without it. I will make a change in this PR (it's not much).